.github/workflows/find-new-flaky-tests.yml

```diff
@@ -61,7 +61,7 @@ on:
 env:
   GIT_HEAD_REF: ${{ inputs.headRef || github.ref }}
   # Exclude paths other than main go project
-  EXCLUDE_PATHS: "integration-tests,dashboard-lib,chainlink-relay,tools/goreleaser-config"
+  EXCLUDE_PATHS: "integration-tests,dashboard-lib,chainlink-relay,tools/goreleaser-config,deployment"
```

Comment:
What is `chainlink-relay` doing here? That is an old module name (renamed to chainlink-common), but it is a separate repo

Reply:
no idea question for @lukaszcl

Reply:
@jmank88 @connorwstein I think we can remove it then.

Reply:
Do we not want to find flaky tests in /deployment?

Reply:
@cgruber we will soon. This flaky detector is a new thing and /deployment will be supported today or early next week.

Reply:
Btw, I removed chainlink-relay from `EXCLUDES_PATHS`